src/components/basic_table/in_memory_table.a11y.tsx

```diff
@@ -212,3 +215,114 @@ describe('EuiInMemoryTable', () => {
     });
   });
 });
+
+describe('EuiTable', () => {
+  describe('Basic table', () => {
```

Comment:
Sorry, to be clear, I'm asking us to keep each major component in their own separate `.a11y.tsx` file. So because `EuiBasicTable` is a completely separate component from `EuiInMemoryTable` (and also the underlying `EuiTable` components), I want us to give it its own `src/components/basic_table/basic_table.a11y.tsx` file and move this describe section to that file, separate from the in memory table tests.

```suggestion
describe('EuiBasicTable', () => {
```

I realize that this adds/potentially duplicates some setup cruft (which you could get around by doing an export/import if you really wanted), but in this case I think that tradeoff is well worth it in order to have separation of concerns and the ability to write more detailed tests for each individual component and expand said tests without starting to stray into 500-1000+ line file territory (which no one likes having to maintain or read).

Hope that makes sense - lmk if not!

Reply:
Alright, I think I've got it bracketed. I took the block you suggested and parked it in a `basic_table.a11y.tsx` file. I opted not to import/export the common `User` type and column generator. It's a bit of duplicated setup, but each test is batteries included this way. I'll take a bit of redundancy in the name of compartmentalization.

Reply:
Perfection and beauty!! The new basic table file feels super easy to reason about and jump from section to section. Thank you!